No Favicon in WP Admin (#483)
humanmade/altis-cms: docs/branding.md

	);
}
```

## Custom Favicon

> The favicon shown on the CMS can be set using wither the CMS or Configuration option. Using the CMS option will override the Configuration option.

Reviewer: ```suggestion
The favicon shown for the application can be set using either the CMS or a configuration option. Using the CMS option will override the configuration option.
```

> #### CMS option

Reviewer: The previous heading level was 2 so this should be 3:
```suggestion
### CMS Option
```

1. Login to the admin area
2. Go to <strong>Appearance</strong> > <strong>Customize</strong> and click on the <strong>Site Identity</strong> tab

> 3. Scroll to <strong>Site Icon</strong> click on the select site icon button and then upload the image you want to use as the icon.

Reviewer: HTML in markdown is usually a last resort for unsupported or complex markup, use `**` to denote bold text instead.

https://www.markdownguide.org/basic-syntax/

> 3. Scroll to **Site Icon** click on the select site icon button and then upload the image you want to use as the icon.
> 4. Make any adjustment to the image if it's too large.

Reviewer: ```suggestion
3. Scroll to **Site Icon** and click on the select site icon button, then upload the image you want to use as the icon.
4. Make any adjustments needed to the image if it's too large.
```

5. Publish your changes.

> #### Configuration option

Reviewer: ```suggestion
### Configuration Option
```

> You can use the `favicon-icon` configuration option to provide a project root relative path to an image file.

Reviewer: ```suggestion
You can use the `favicon` configuration option to provide a project root relative path to an image file.
```

```json
{
	"extra": {
		"altis": {
			"modules": {
				"cms": {

> 					"favicon-icon": "/content/favicon.png"

Reviewer: ```suggestion
					"favicon": "/content/favicon.png"
```
